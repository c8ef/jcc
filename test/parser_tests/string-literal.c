int main(void) {
  const char* str = "Hello, world!";
  const char a = 'a';
  return 0;
}
