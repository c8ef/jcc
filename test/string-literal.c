int main(void) {
  const char* str = "Hello, world!";
  return 0;
}
