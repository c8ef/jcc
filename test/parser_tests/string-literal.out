FunctionDecl: main
  Args[0]
  CompoundStatement
    DeclStatement
      VarDecl: str
        StringLiteral: Hello, world!
    DeclStatement
      VarDecl: a
        CharacterLiteral: a
    ReturnStatement
      IntergerLiteral: 0
